### ISSUE TITLE
Set out of office message

### ISSUE THREAD

**Issue by lodesmets** [RogerSelwyn/O365-HomeAssistant#76]
Hey,
I was looking through the office api, and I notice it is possible to set the out of office message (auto responder)
[examples](https://gotoguy.blog/2019/07/11/get-and-set-automatic-replies-like-oof-with-microsoft-graph/)
[docs](https://learn.microsoft.com/en-us/graph/api/resources/automaticrepliessetting?view=graph-rest-1.0)

I will take a look at it myself, and try to see if I can add it. But I'm currently reading the code, to see how this package works :)

**Comment by RogerSelwyn** [comment]
Feel free to raise a PR. Probably best as a service, would need to decide on where to locate it. Possibly in a new class file.

Most of the code I've adopted from the previous maintainer, but changed and extended to meet new needs.

**Comment by lodesmets** [comment]
I made the pull request to the O365 library, once that is approved (and released), I will take a look into adding it here

**Comment by RogerSelwyn** [comment]
Great stuff 😀

**Comment by lodesmets** [comment]
It has been merged into the python library. I'm looking to add it to this project. 
I'm going to try and add an set_auto_reply service

**Comment by RogerSelwyn** [comment]
Thanks. I think it needs to support add/change/delete.

**Comment by RogerSelwyn** [comment]
Included with - https://github.com/RogerSelwyn/O365-HomeAssistant/releases/tag/v4.0.4
